library/common/types/utility.cc

```diff
@@ -0,0 +1,27 @@
+#include <string>
+
+#include "library/common/data/utility.h"
+
+namespace Envoy {
+namespace Types {
```

Comment:
I'd suggest the namespace "Bridge" here.

Reply:
Done in https://github.com/envoyproxy/envoy-mobile/pull/1602/commits/2f00eae81101ae1b94c16e5f15ee84d78bfed3a5